### PR TITLE
Notify on failure if attempting to deploy the image marked as latest

### PR DESCRIPTION
**Pull request by ThomasOSI**
@pemaOSI Ready for review

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,4 +1,8 @@
 node {
+    def curStage = 'Start'
+    def emailList = EMAIL_NOTIFICATION_LIST ?: 'thomas.ramirez@osi.ca.gov'
+    def branch = env.BRANCH_NAME ?: 'master'
+
```
Comment (pemaOSI) [inline Jenkinsfile]: You could set DOCKER_IMAGE default value to be "latest"

**Comment by pemaOSI** [comment]
looks good

File: Jenkinsfile (hunk 1 above)
Comment (ThomasOSI) [inline Jenkinsfile]: This Jenkinsfile is used by both the nightly and CI deployments so I have set the default image to "latest" within the nightly deployment pipeline configurations

Comment (pemaOSI) [inline Jenkinsfile]: Lets be concise with terms. For instance, "latest" is image tag while image is different from tag.

**Pull request closed without merging**